### DISCUSSION TITLE
OrchardCore.MiniProfiler not showing up on the pages

### DISCUSSION THREAD

**Discussion by DotCat1985** [OrchardCMS/OrchardCore#8380] [Q&A]
Hi,
I activated `OrchardCore.MiniProfiler` by OC dashboard, but it's not showing up on the pages.
My web app is connected to the SQLite db.
Have I forgottern something?

**Comment by jtkech** [comment]
Just used it for some tests ;)

It works, by default it only shows on the front end, there is an option to show on the admin, i don't think this option is configurable through the admin UI, i updated the code for my tests.

**Reply by DotCat1985** [reply]
OK. The issue was due to missing `@await RenderSectionAsync("Footer", required: false)` in the footer of `Layout` view.
So I solved it and thank you :)
